project/Settings.scala

```diff
@@ -57,7 +57,7 @@ object Settings {
         from("openjdk:8")
         expose(8000)
         add(artifact, artifactTargetPath)
-        entryPoint("java", "-jar", artifactTargetPath)
+        entryPoint("/bin/bash", "-c", "java -jar " + artifactTargetPath + " $CROMWELL_ARGS")
```

Comment:
note: the /bin/bash is because otherwise the variable expansion won't happen (thanks Docker!)